__tests__/app.test.js

```diff
@@ -11,10 +11,21 @@ beforeEach(() => {
 afterAll(() => db.end());
 
 describe("app", () => {
+	describe("Global Error Handling Tests", () => {
+		test("Status: 404, responds with 'Bad Request - Path Not Found'", () => {
```

Comment:
It's a bit fussy but 404 is just Not Found, Bad Request is for status 400! Check http.cat or http.dog if you're ever not fully sure!

Reply:
Thanks!